src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -80,7 +80,7 @@ interface Props extends RelayProps {
   conversationId: string
 }
 
-const track: Track<Props, null, Schema.Entity> = _track
+const track: Track<Props, null> = _track
```

Comment:
Now that you’ve made `null` the default for `S`, you can remove these as well.

Reply:
👌